doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md

```diff
@@ -124,3 +124,21 @@ Install facedetect for [image thumbnails focal point](../../04_Assets/03_Working
 ```bash
 apt-get install facedetect
 ```
+
+## html2text
```

Comment:
not sure if merge conflicts resolved correctly. html2text section should be removed from here. please see https://github.com/pimcore/pimcore/pull/6448/files#diff-8c29cd6e0351c1a7c57ad1524fd58eaf

Reply:
fixed! ;)